### PR TITLE
Refactor tests #2

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
- Clean up `confest.py`
- Move test related to `importers` to `tests/importers`
- Move tests related to `model` and `data` to `tests/test_model.py` and `tests/test_data.py`
- Added some more tests for `model` and `data` 

(This PR is part of multiple PRs to refactor our tests. We want to have one test file per python file so that tests can be found more easily and we avoid code duplication (e.g. for helper functions).)

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by federicotdn** [comment]
Do we know what's making the build fail? It's stopping mid-testing on Python 3.7 only, without any error messages. I've restarted the build once and the same thing happened again.

**Comment by tabergma** [comment]
mmhh... not sure, will take a closer look

**Comment by tabergma** [comment]
@federicotdn I'm not sure how to solve this. Tried to reproduce it locally - failed. Tried to get an error message on travis using the debug mode - not successful. Do you have any other idea? 
It seems to be always the same test: `test_train_model_on_test_pipelines`. python3.7 failed every time on that test without any error and python3.5 from time to time.

**Comment by federicotdn** [comment]
@tabergma I'll check this out today! I suspect it could be because of the [OOM killer](https://www.memset.com/docs/additional-information/oom-killer/), but I'm not sure (it would explain at least why it fails on Travis and not locally).

File: Makefile [hunk 1]
```diff
@@ -57,7 +57,12 @@ prepare-tests-files:
 	wget --progress=dot:giga -N -P data/ https://s3-eu-west-1.amazonaws.com/mitie/total_word_feature_extractor.dat
 
 test: clean
-	py.test tests --cov rasa
```
Comment (federicotdn) [inline Makefile]: If we changed this back to the way it was, do the tests still continue to pass? I know separating the tests into various invocations of pytest works, but I don't think it's as practical as just doing `pytest tests/`. If running just that command alone causes Travis to fail then I'd say it's still possible we have a problem regarding memory usage (given that running the tests using 6 separate processes works OK).

Comment (tabergma) [reply Makefile]: Still on my todo list. Will test it in the afternoon. As soon as I have some results, I'll come back to you.

**Comment by tabergma** [comment]
@federicotdn I cannot find the root cause. I guess, the best thing to do is, start again from scratch and check after each step if the memory consumption increases or not. Will close this PR for now as it is not worth spending more time on it. Thanks for your help!

**Pull request closed without merging**